test/Google.Api.Gax.Tests/RetryTest.cs

```diff
@@ -31,8 +32,30 @@ public class RetryTest
             MaxDelay = TimeSpan.FromTicks(6000) // Irrelevant
         };
 
-        [Fact]
-        public void FirstCallSucceeds()
+        private static Task<TResponse> Call<TRequest, TResponse>(
```

Comment:
Again, I'd avoid calling the parameter async if possible. (Even just `asynchronous` would be better.) Maybe I'm being too picky though.